src/ddscxx/include/dds/topic/TBuiltinTopicKey.hpp

```diff
@@ -43,14 +43,14 @@ class dds::topic::TBuiltinTopicKey : public ::dds::core::Value<D>
      *
      * @return the BuiltinTopicKey
      */
-    const int32_t* value() const;
```

Comment:
Isn't this part of the specification, and therefore should this not be changed?

Reply:
Never mind, after further discussion with @eboasson , it seems that changing this to int8_t will not cause large issues.